tensorflow/compiler/jit/xla_gpu_device.cc

```diff
@@ -52,8 +53,34 @@ Status XlaGpuDeviceFactory::CreateDevices(
     VLOG(1) << "Failed to create XLA_GPU device: " << platform.status();
     return Status::OK();
   }
-
-  for (int i = 0; i < platform.ValueOrDie()->VisibleDeviceCount(); ++i) {
+  const auto& allowed_gpus =
```

Comment:
Perhaps s/`const auto&`/`string`/?  That would make this more clear, I think.

Reply:
Done.